package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "webpack-cordova-plugin",
-  "version": "0.1.5",
+  "name": "webpack-cordova-plugin-fork",
```

Comment:
No problem! I just found it weird that it has been published also
Thanks though!